rllib/agents/dqn/dqn.py

```diff
@@ -153,33 +157,68 @@ def check_config_and_setup_param_noise(config):
                               config.get("n_step", 1))
     config["sample_batch_size"] = adjusted_batch_size
 
+    # Backward compatibility of epsilon-exploration config AND beta-annealing
+    # fraction settings (both based on schedule_max_timesteps, which is
+    # deprecated).
```

Comment:
Add a TODO to remove this at some future time.

Reply:
done